interface/forms/dictation/view.php

```diff
@@ -1,31 +1,61 @@
-<!-- Forms generated from formsWiz -->
 <?php
+/** 
+ *  Dictation Form Edit Saved Data
+ * 
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ * @copyright Copyright (c) 2017 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * 
+ */
 
-
-
-
+use OpenEMR\Core\Header;
 include_once("../../globals.php");
+require_once("../../../public/assets/htmlpurifier-4.9.2/library/HTMLPurifier.auto.php");
```

Comment:
After you rebase code to most recent codebase, then you can remove the above line.